content/benchmarks/_index.md

```diff
@@ -3,4 +3,5 @@ layout: benchmarks
 header: dark
 footer: dark
 title: Haystack Benchmarks
+version: latest
```

Comment:
Can we maybe use a "real" version here? So that we don't need to have a `latest` folder that would be overwritten every time we add a new version

Reply:
Additionally to that, does it make sense to make the version number come up so that we can access the 'real/lates' version via it's version tag too? E.g. now it would be: https://haystack-home-git-benchmark-versions-deepset-overnice.vercel.app/benchmarks/v1.9.0
Which would also be accessible on https://haystack-home-git-benchmark-versions-deepset-overnice.vercel.app/benchmarks (because it has the 'real' tag